js_modules/dagster-ui/packages/ui-core/src/asset-graph/KeyboardTag.tsx

```diff
@@ -7,7 +7,9 @@ interface KeyboardTagProps {
 
 export const KeyboardTag = styled.div<KeyboardTagProps>`
   ${(props) => {
-    return props.$withinTooltip ? `color: ${Colors.accentWhite()}` : `color: ${Colors.textLight()}`;
+    return props.$withinTooltip
+      ? `color: ${Colors.accentWhite()}`
+      : `color: ${Colors.textDefault()}`;
   }};
```

Comment:
to fix the shortcut text color
<img width="443" height="225" alt="Screenshot 2025-08-29 at 4 29 00 PM" src="https://github.com/user-attachments/assets/e4959fc7-1972-4162-8a9f-48261c2e1cc0" />